### PR TITLE
Indexing equipments in elasticsearch in parallel with flushing netwok and sending reports

### PR DESCRIPTION
**Pull request by SlimaneAmar**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
No Indexing equipments in Elasticsearch when network import

**What is the new behavior (if this is a feature change)?**
Indexing equipments when network import
Flushing network, sending reports and indexing in parallel

### CODE REVIEW

File: src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java [hunk 1]
```diff
@@ -219,7 +279,32 @@ NetworkInfos importCgmesCase(UUID caseUuid, List<BoundaryInfos> boundaries) {
...
+            LOGGER.trace("Flush network '{}' in parallel : {} seconds", networkUuid, TimeUnit.NANOSECONDS.toSeconds(System.nanoTime() - startTime.get()));
+        }
+    }
+
+    private void insertEquipmentIndexes(Network network, UUID networkUuid) {
+        AtomicReference<Long> startTime = new AtomicReference<>(System.nanoTime());
+        try {
+            network.getIdentifiables()
+                    .stream()
+                    //.filter(Predicate.not(i -> i instanceof Switch))
```
Comment (geofjamg) [inline src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java:296]: To clean

File: src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java [hunk 2]
```diff
@@ -219,7 +279,32 @@ NetworkInfos importCgmesCase(UUID caseUuid, List<BoundaryInfos> boundaries) {
...
+                    .map(c -> toEquipmentInfos(c, networkUuid))
+                    .collect(Collectors.groupingBy(EquipmentInfos::getEquipmentType)).values()
+                    .parallelStream()
```
Comment (geofjamg) [inline src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java:299]: Is it really impacting performance?

File: src/main/java/com/powsybl/network/conversion/server/elasticsearch/EquipmentInfosServiceMock.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/*
+  Copyright (c) 2020, RTE (http://www.rte-france.com)
```
Comment (geofjamg) [inline src/main/java/com/powsybl/network/conversion/server/elasticsearch/EquipmentInfosServiceMock.java:2]: 2021

**Review by geofjamg — CHANGES_REQUESTED** [review]
Just minors cleanup!

File: src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java (hunk 1 above)
Comment (SlimaneAmar) [reply src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java:296]: Done

File: src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java (hunk 2 above)
Comment (SlimaneAmar) [reply src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java:299]: Now with indexation in parallel no

File: src/main/java/com/powsybl/network/conversion/server/elasticsearch/EquipmentInfosServiceMock.java (hunk 3 above)
Comment (SlimaneAmar) [reply src/main/java/com/powsybl/network/conversion/server/elasticsearch/EquipmentInfosServiceMock.java:2]: Done

**Review by geofjamg — APPROVED** [review]

**Pull request merged**